package.json

```diff
@@ -57,7 +61,21 @@
         "node": ">=4.2.1"
     },
     "scripts": {
-        "test": "node tests/all-tests.js",
+        "pretest": "npm run pretest:clean && npm run pretest:instrument && npm run pretest:copyUninstrumentedCode",
+        "pretest:clean": "node node_modules/rimraf/bin.js reports/* coverage/* instrumented/*",
```

Comment:
We shouldn't clean up `report.tap`, as it is evidence of the last run that's used by the QI system, and needs to stick around.  `.nyc_output` is a good catch, I have added that to the cleanup.